src/Library/demos/Breakpoints/main.blp

```diff
@@ -0,0 +1,57 @@
+using Gtk 4.0;
+using Adw 1;
+
+Gtk.Window {
+  width-request: 360;
+  height-request: 200;
+  default-width: 640;
+  default-height: 480;
+  title: _("Breakpoints");
+
+  Box {
+    orientation: vertical;
+    spacing: 12;
+    valign: center;
+
+    Image image {
+      icon-size: large;
+      icon-name: "face-devilish-symbolic";
```

Comment:
Could you replace this with something more culturally neutral? 

the smiley should be fine